app_test.go

```
@@ -879,46 +880,103 @@
 
-	go func() {
-		err := app.Listener(ln)
-		assert.NoError(t, err)
-	}()
+		ln := fasthttputil.NewInmemoryListener()
+		serverStarted := make(chan bool, 1)
 
-	time.Sleep(1 * time.Second)
+		go func() {
+			serverStarted <- true
+			_ = app.Listener(ln)
```

golangci-lint (1.62.2)

Error return value of `app.Listener` is not checked

(errcheck)